addons/core/translations/form/en-us.yaml

```diff
@@ -115,3 +115,12 @@ value:
 callback_url:
   label: Callback URL
   help: The URL to which the third-party provider sends the authentication response.  Paste this into the provider configuration.
+vault_path:
+  label: Vault Path
+  help: The path in Vault to request credentials from.
```

Comment:
This seems fine because of the field name.

Reply:
Did this description come from the Go code?  They usually include decent descriptions.

Reply:
From credential library [docs](https://www.boundaryproject.io/docs/concepts/domain-model/credential-libraries#vault-credential-library-attributes) on boundaryproject.io :).

Reply:
Updated description to `The path the library uses when requesting credentials from Vault.`